lib/components/app/app-menu.tsx

```diff
@@ -176,85 +138,80 @@ class AppMenu extends Component<
 
     const { isPaneOpen } = this.state
     const { SvgIcon } = this.context
+    const buttonLabel = isPaneOpen
+      ? intl.formatMessage({ id: 'components.AppMenu.closeMenu' })
+      : intl.formatMessage({ id: 'components.AppMenu.openMenu' })
+
     return (
       <>
-        <div
-          aria-label={
-            isPaneOpen
-              ? intl.formatMessage({ id: 'components.AppMenu.closeMenu' })
-              : intl.formatMessage({ id: 'components.AppMenu.openMenu' })
-          }
-          className="app-menu-icon"
+        <button
+          aria-controls="app-menu"
+          aria-expanded={isPaneOpen}
+          aria-label={buttonLabel}
+          className={`app-menu-icon ${isPaneOpen ? 'open' : ''}`}
           onClick={this._togglePane}
-          onKeyDown={this._togglePane}
-          role="button"
-          tabIndex={0}
         >
-          <span className={isPaneOpen ? 'menu-left-x' : 'menu-top-line'} />
-          <span className={isPaneOpen ? '' : 'menu-middle-line'} />
-          <span className={isPaneOpen ? 'menu-right-x' : 'menu-bottom-line'} />
-        </div>
+          <span />
+          <span />
+          <span />
+        </button>
+        {/* Use a button for skipping navigation. A regular <a> element would modify the URL,
+            and such change would be captured by the router without changing the focused element. */}
+        <button
+          className="skip-nav-button"
+          onClick={this._handleSkipNavigation}
+        >
+          <FormattedMessage id="components.AppMenu.skipNavigation" />
+        </button>
         <SlidingPane
           from="left"
+          hideHeader
           isOpen={isPaneOpen}
           onRequestClose={this._togglePane}
+          shouldCloseOnEsc
           width="320px"
         >
-          <div className="app-menu">
+          <div className="app-menu" id="app-menu">
             {/* This item is duplicated by the view-switcher, but only shown on mobile
             when the view switcher isn't shown (using css) */}
-            <MenuItem
+            <AppMenuItem
               className="app-menu-route-viewer-link"
```

Comment:
I don't think this className is getting properly applied -- `app-menu-route-viewer-link` should disappear on >768px screen sizes but the rule doesn't seem to be applying.

Reply:
Fixed in https://github.com/opentripplanner/otp-react-redux/pull/749/commits/2434a1c3b347761d8228160badf161dced0dbc18. The `display` attribute I added to the buttons conflicted with the one that sets visibility for the route viewer menu.